### PR TITLE
Quickstart improvements

### PR DESCRIPTION
**Pull request by jgiannuzzi**
* Fix Prometheus installation warnings
* Make it work on macOS and Windows
* Remove obsolete and duplicate commands
* Do not rely on the Docker host IP

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -3,14 +3,60 @@
...
+
+You can install the pre-requisites with [Chocolatey](https://chocolatey.org):
+
+```cmd
+choco install git docker-desktop kubernetes-helm kind kubernetes-cli
+```
+
+Ensure at least 5GB of RAM are allocated to the Docker VM (see Settings -> Resources -> Advanced).
+
+In order to follow the instructions below as-is, you should also install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about).
```
Comment (jankaspar) [inline docs/quickstart.md]: I wonder if this would all just work with bash on windows (it is usually installed with git).

Comment (jgiannuzzi) [reply docs/quickstart.md]: Good point, I'll try it out.

Comment (jgiannuzzi) [reply docs/quickstart.md]: Indeed, it works fine with Git Bash, I just had to adapt the downloading of `armadactl`

**Comment by jgiannuzzi** [comment]
@jankaspar I dropped the WSL requirement. Can you please review again? Thanks!

**Pull request merged**